week-3/A-array-find/exercise.js

```diff
@@ -1,13 +1,17 @@
-/* 
-  You are given an array of test scores.
-  Any result over 60 is a pass. 
-  Using .find() (and any other array methods necessary), find the lowest pass score
+/*
+  You are given an array of names.
+  Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 */
 
-var testScores = [90, 50, 100, 25, 81, 66, 80];
-var lowestPassScore = findLowestPassScore(testScores);
+// write your code here
+function findLongNameThatStartsWithA(name) {
+  return name.length > 7 && name[0] === "A";
```

Comment:
Nice work :)

There is a little problem though. What happens if we change the `names` array to have "alexandra" instead of "Alexandra"? Is our answer still correct? How could we fix it?